cmd/parse-host-file/parse-host-file.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+type host struct {
+	DNS string `json:"dns_name"`
+	ID  string `json:"instance_id"`
+}
+
+func main() {
+	var hostFile string
+	flag.StringVar(&hostFile, "file", "", "path to the file containing spawned host info")
+	flag.Parse()
+
+	if hostFile == "" {
+		fmt.Println("no host file provided")
+		return
+	}
+
+	f, err := os.Open(hostFile)
+	if err != nil {
+		fmt.Println(err.Error())
+		return
+	}
+	bytes, err := ioutil.ReadAll(f)
+	if err != nil {
+		fmt.Println(err.Error())
+		return
+	}
+	var hosts []host
+	err = json.Unmarshal(bytes, &hosts)
+	if err != nil {
+		fmt.Println(err.Error())
+		return
+	}
+	if len(hosts) < 1 {
+		fmt.Println("no hosts listed in file")
+		return
```

Comment:
use grip.EmergencyFatal() to get exit codes for all of the early returns.

Reply:
I think without calling something to exit with a non-zero code, you'll endup always exiting 0, which doesn't make sense.

Reply:
added explicit exits for error conditions